config/routes.rb

```diff
@@ -10,6 +10,9 @@
     collection do
       post 'solicitors_search'
     end
+    member do
+      put  'close'
+    end
```

Comment:
There are going to be a bunch of different actions on defence requests, so it makes sense to use a block I think?